### PR TITLE
Support \widecheck

### LINKED ISSUE #1221: support \widecheck
`\widecheck` is a wide `\check` and an upside-down `\widehat`, which is available in packages `mathabx` and `stix`.

### PR DESCRIPTION
**Pull request by ronkok**
Fixes #1221.

Implemented as an upside-down version of `\widehat`.

### CODE REVIEW

**Comment by ronkok** [comment]
Here's how they look:

wck

File: src/svgGeometry.js [hunk 1]
```diff
@@ -281,6 +281,19 @@ c-1 5-5 9-11 9h-2L532 67 19 159h-2c-5 0-9-4-11-9l-5-22c-1-6 2-12 8-13z`,
     widehat4: `M1181 0h2l1171 296c6 0 10 5 10 11l-2 23c-1 6-5 10
 -11 10h-1L1182 67 15 340h-1c-6 0-10-4-11-10l-2-23c-1-6 4-11 10-11z`,
 
+    // widecheck paths are all inverted versions of widehat
```
Comment (kevinbarabash) [inline src/svgGeometry.js]: An optimization we might want to consider is using an SVG transform to invert the existing geometry from `\widehat`.

Comment (ronkok) [reply src/svgGeometry.js]: I thought about that, but I've already got so many exceptions in this code that I wanted to add something without a whole new rendering method.

**Comment by ronkok** [comment]
Closing for repository repair. Will reopen shortly.

**Comment by kevinbarabash** [comment]
`unknown repository` eek

**Comment by ronkok** [comment]
Sorry. I ran `git stash apply` while in the wrong branch and thoroughly tangled things up. So I've completely over written that repository with a replacement. The `widecheck` work is now in PR #1406.

**Pull request closed without merging**